src/components/network/filter-creation-panel.tsx

```diff
@@ -28,11 +29,8 @@ import {
 import { fetchElementsMetadata } from 'services/explore';
 import { UniqueNameInput } from 'components/dialogs/commons/unique-name-input';
 import { useSelector } from 'react-redux';
-import {
-    equipementTypeToLabel,
-    EQUIPMENT_TYPES,
-} from '../utils/equipment-types';
 import { UUID } from 'crypto';
+import { FormEquipment } from '@gridsuite/commons-ui/dist/components/filter/utils/filter-form-utils';
```

Comment:
i think it not normal to include components from @gridsuite/commons-ui/**dist/**

Reply:
Indeed. I simply switched it to `from '@gridsuite/commons-ui';`. Webstorm complains but it works fine.

I guess I miss some kind of config for the auto import.